elephant/test/test_spike_train_generation.py

```diff
@@ -16,7 +16,7 @@
 from numpy.testing.utils import assert_array_almost_equal
 from scipy.stats import kstest, expon
 from quantities import ms, second, Hz, kHz, mV, dimensionless
-
+import matplotlib.pyplot as plt
```

Comment:
Matplotlib is not used within Elephant atm. If you remove this import than the unittests should work again.